main.css

```diff
@@ -84,3 +84,12 @@ input[type="number"] {
 		transform:rotate(10deg)
 	}
 }
+
+#error-message{
+  color: #D8000C;
+  animation: fadeIn 5s;
```

Comment:
5s is too long for a transition. Try using 1s or 500ms instead.

Reply:
commited your suggested changes.

Reply:
The declarations are outside the rule-set.
Also, the color is not included in the new rules